pipelines/pingcap/tiflash/latest/pull_integration_test.groovy

```diff
@@ -61,6 +64,9 @@ pipeline {
                     try {
                         dir("test-build-cache") {
                             cache(path: "./", includes: '**/*', key: prow.getCacheKey('tiflash', REFS, 'it-build')){
+                                if (REFS.base_ref.startsWith(next_gen_branch_prefix)) {
+                                    is_branch_next_gen = true
+                                }
```

Comment:
The changes in this PR works under "feature/next-gen" branch. But if we later merge the feature branch into master, can we run "integration_test" under "classic build" and "next-gen build" in the same PR?

Reply:
Will the situation of simultaneously building classic builds and next-gen builds exist in the long term?

Reply:
If this situation persists for a long time, it might be more appropriate to create a new pull-integration-next-gen.

Reply:
Yes, I think the situation of simultaneously building classic builds and next-gen builds exist in the long term. It is better to create a new pull-integration-next-gen.

What's more, not only the "pull-integration-test" but also the "pull-unit-test" need a new next-gen testing. Because different compile option related to different unit-test logic and unit-test case.

Reply:
Understood, it seems that adding pull-integration-next-gen and pull-unit-next-gen is a better choice.